LightAPRS-hab/LightAPRS-hab.ino

```diff
@@ -15,6 +15,17 @@
 #define PIN_DRA_RX  22
 #define PIN_DRA_TX  23
 
+#define RedLedPin   7
```

Comment:
I think it makes sense to put these in the config file, just because they're defined by the hardware surrounding the LightAPRS and not the LightAPRS itself.

Reply:
Maybe also add some comments that mention what they signify too?